public/docs/dart/latest/quickstart.jade

```diff
@@ -21,38 +21,40 @@ p.
     Create a new directory, and put a file named
     <code>pubspec.yaml</code> in it.
 
-  pre.prettyprint
-    code.
-      &gt; mkdir hello_world
-      &gt; cd hello_world
-      &gt; vim pubspec.yaml  # Use your favorite editor!
+  code-example(language="sh").
+    &gt; <span class="blk">mkdir hello_world</span>
```

Comment:
It's not very noticeable on my screen, but I wanted to highlight the text they should enter, without using the pink highlight I used in code.